core/src/main/java/hudson/lifecycle/WindowsInstallerLink.java

```diff
@@ -310,6 +311,8 @@ static int runElevated(File jenkinsExe, String command, TaskListener out, File p
         } finally {
             try (InputStream fin = Files.newInputStream(new File(pwd,"redirect.log").toPath())) {
                 IOUtils.copy(fin, out.getLogger());
+            } catch (InvalidPathException e) {
+                // ignore;
```

Comment:
Well if `new File(pwd, "redirect.log")` is an invalid path, there is no redirect log to copy to out.getLogger() so it seems somewhat silly to log the invalid path... the log message would be "We couldn't copy the redirected logs because the redirected logs were not stored on a valid path and so we couldn't ready them back"... the reality is if we hit that exception there are no logs to copy and hence squash the exception. Something else should be catching the invalid path (probably the thing that threw the exception we are in a finally block because of)